src/main/webapp/resources/js/components/pipeline/Pipeline.jsx

```diff
@@ -88,7 +88,10 @@ export default class Pipeline extends React.Component {
                   href={pipelineUrl}
                 >
                   {i18n("pipelines.cart.select")}
-                </Button>
+                </Button>,
+              <Button type="link" href={setBaseUrl(`launch?id=${this.props.id}`)}>
```

Comment:
This is only temporary so I can compare between the two

Reply:
Sounds good